aodncore/pipeline/handlerbase.py

```diff
@@ -609,20 +622,51 @@ def temp_dir(self):
         if self._instance_working_directory:
             return os.path.join(self._instance_working_directory, 'temp')
 
+    #
+    # private properties
+    #
+
+    @property
+    def _upload_runner(self):
+        """Private read-only property for accessing the instance 'upload' upload runner (for internal use only)
+
+        :return: :py:class:`StoreRunner`
+        """
+        if self._upload_runner_object is None:
+            self._upload_runner_object = get_store_runner(self._config.pipeline_config['global']['upload_uri'],
+                                                          self._config, self.logger)
+            self.logger.sysinfo("get_store_runner (upload)-> {runner}(broker='{broker}')".format(
+                runner=self._upload_runner_object.__class__.__name__, broker=self._upload_runner_object.broker.__class__.__name__))
+        return self._upload_runner_object
+
+    @property
+    def _upload_runner_archive(self):
```

Comment:
For simplicity and consistency, could this just be called `_archive_runner`?

Reply:
Yes good point, this is a relic of before this was refactored into the current store runner/broker design.

I've updated to reflect what they actually represent now!